tutorials/shield_multiphysics/doc/content/user_workshop/problem/step02.md

```diff
@@ -0,0 +1,56 @@
+# Step 2: Simple Heat Conduction Kernel id=step02
+
+!---
+
+## Implementing equations in MOOSE
+
+To implement the Heat Conduction equation, we will use a [Kernel](Kernels/index.md) object to compute the
+residual and the Jacobian of the diffusion term:
+
+!equation
+-\nabla \cdot k \nabla T = 0,
+
+where  $k$  is the thermal diffusivity.
+
+A `Kernel` is C++ class, which inherits from `MooseObject` that is used by MOOSE for coding
+volume integrals of a [!ac](PDE).
+
+!!end-intro
+
+!---
+
```

Comment:
In the presentation, there is a lot of content between these 2 slides. Could be nice to have a reminder. Also suggest summarizing the values of the diffusion coefficient on each block, maybe with a picture?

Reply:
Ok added a reminder.